crates/cli/src/args.rs

```diff
@@ -0,0 +1,41 @@
+//! This file contains the structure of command line arguments supported by the
```

Comment:
This whole file has insufficient comments. You have to remember that clap uses the comments on the parts for the help output, so please change this to provide actually useful output when the user calls `hieratika --help`.

Reply:
![image](https://github.com/user-attachments/assets/5e625188-58e5-44c9-8657-160c76936642)

Yes, the lines generated in the help menu were bad. Is it a better output now?